spec/spec_helper.rb

```diff
@@ -91,6 +91,8 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  require "rack_session_access/capybara"
```

Comment:
I would create a new rack_session file in the support folder and put this line there